app/backend/Controllers/CreditDataController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using backend.Exceptions;
+using Newtonsoft.Json;
+using backend.Models.CreditDataModels;
+
+namespace backend.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class CreditDataController : Controller
+    {
+        private const string devSkillsApi = "https://infra.devskills.app/api";
```

Comment:
I would prefer to pass this in via the pipeline, so this would become a part of the settings making it available for future APIs